.drone/drone.yml

```
@@ -17,7 +17,7 @@ steps:
 
 # binaries
 - name: build-tempo-binaries
-  image: golang:1.16.0-alpine
+  image: golang:1.17.0-alpine
```

1.17.1 is already released. Should we just track latest 1.17?
```suggestion
  image: golang:1.17-alpine
```